test/infrastructure/inmemory/internal/cloud/api/v1alpha1/etcdcluster_annotations.go

```diff
@@ -34,4 +34,7 @@ const (
 	// By using this mechanism leadership can be forwarded to another pod with an atomic operation
 	// (add/update of the annotation to the pod/etcd member we are forwarding leadership to).
 	EtcdLeaderFromAnnotationName = "etcd.inmemory.infrastructure.cluster.x-k8s.io/leader-from"
+
+	// EtcdMemberRemoved is added to etcd pods which have been removed from the etcd cluster.
+	EtcdMemberRemoved = "etcd.inmemory.infrastructure.cluster.x-k8s.io/removed"
```

Comment:
```suggestion
	EtcdMemberRemoved = "etcd.inmemory.infrastructure.cluster.x-k8s.io/member-removed"
```
nit: maybe, up to you (seems more consistent with the two annotations above)